src/components/SignUp/OrganizationInformation.tsx

```diff
@@ -61,6 +63,24 @@ class OrganizationInformation extends Component<Props, State, {}> {
     };
   }
 
+  ConvertNumber(number, format): string {
```

Comment:
jw where are we using this function? also can we have some tests around this?

Reply:
this was the first method i tried, but i will delete this!